app/api/ui/menu.js

```diff
@@ -1,162 +1,181 @@
       {
         key: 'tables',
-        name: 'Table',
+        name: ' Your Portfolio',
         icon: 'grid_on',
         link: '/app/pages/table'
       },
-      {
-        key: 'maintenance',
-        name: 'Maintenance',
-        icon: 'settings',
-        link: '/maintenance'
-      },
-      {
-        key: 'coming_soon',
-        name: 'Coming Soon',
-        icon: 'polymer',
-        link: '/coming-soon'
-      },
+      // {
+      //   key: 'maintenance',
+      //   name: 'Maintenance',
+      //   icon: 'settings',
+      //   link: '/maintenance'
+      // },
+      // {
+      //   key: 'coming_soon',
+      //   name: 'Coming Soon',
+      //   icon: 'polymer',
+      //   link: '/coming-soon'
+      // },
     ]
   },
-  {
-    key: 'errors',
-    name: 'Erros',
-    icon: 'pets',
-    child: [
-      {
-        key: 'errors',
-        name: 'Errors',
-        title: true,
-      },
-      {
-        key: 'not_found_page',
-        name: 'Not Found Page',
-        icon: 'pets',
-        link: '/app/pages/not-found'
-      },
-      {
-        key: 'error_page',
-        name: 'Error Page',
-        icon: 'flash_on',
-        link: '/app/pages/error'
-      },
-    ]
-  },
-  {
-    key: 'menu_levels',
-    name: 'Menu Levels',
-    multilevel: true,
-    icon: 'sort',
-    child: [
-      {
-        key: 'level_1',
-        name: 'Level 1',
-        link: '/#'
-      },
-      {
-        key: 'level_2',
-        keyParent: 'menu_levels',
-        name: 'Level 2',
-        child: [
-          {
-            key: 'sub_menu_1',
-            name: 'Sub Menu 1',
-            link: '/#'
-          },
-          {
-            key: 'sub_menu_2',
-            name: 'Sub Menu 2',
-            link: '/#'
-          },
-        ]
-      },
-    ]
-  }
+  // {
+  //   key: 'errors',
+  //   name: 'Erros',
+  //   icon: 'pets',
+  //   child: [
+  //     {
+  //       key: 'errors',
+  //       name: 'Errors',
+  //       title: true,
+  //     },
+  //     {
+  //       key: 'not_found_page',
+  //       name: 'Not Found Page',
+  //       icon: 'pets',
+  //       link: '/app/pages/not-found'
+  //     },
+  //     {
+  //       key: 'error_page',
+  //       name: 'Error Page',
+  //       icon: 'flash_on',
+  //       link: '/app/pages/error'
+  //     },
+  //   ]
+  // },
+  // {
+  //   key: 'menu_levels',
+  //   name: 'Menu Levels',
+  //   multilevel: true,
+  //   icon: 'sort',
+  //   child: [
+  //     {
+  //       key: 'level_1',
+  //       name: 'Level 1',
+  //       link: '/#'
+  //     },
+  //     {
+  //       key: 'level_2',
+  //       keyParent: 'menu_levels',
+  //       name: 'Level 2',
+  //       child: [
+  //         {
+  //           key: 'sub_menu_1',
+  //           name: 'Sub Menu 1',
+  //           link: '/#'
+  //         },
+  //         {
+  //           key: 'sub_menu_2',
+  //           name: 'Sub Menu 2',
+  //           link: '/#'
+  //         },
+  //       ]
+  //     },
+  //   ]
+  // }
```

Comment:
Ideally we want to keep as little commented code on production ready products, so when committing something to one of the main branches, we want to always be sure to not leave commented code (there are a few rare exceptions where we don't do this, but this case isn't one of them)

Either way, your git log should have every state your code as been in, so if you ever need to recover some previous existing code you can just check the history and get it from there, instead of leaving clutter in your code.